builder/views.py

```diff
@@ -60,6 +63,15 @@ def _draft(request, draft, search_slug):
     if request.method == "POST":
         return _handle_post(request, draft)
 
+    # Currently we have no way to record the releases for a code system
+    # At the point of any GET for a draft, we can't tell if the codes may have changed,
+    # so we need to update the draft again.  This will recreate all the searches and
+    # check and update the codeset for any new concepts.
+    # We cache the updated draft ids for 60 mins so this isn't done on every single GET request
```

Comment:
I thought we cached them for 24 hours?

Reply:
I changed my mind about how long to cache for and decided 60 mins was overkill, given that coding systems are updated quite infrequently.  Forgot to update the comment